external/urlfrontier/src/main/java/com/digitalpebble/stormcrawler/urlfrontier/StatusUpdaterBolt.java

```diff
@@ -127,7 +132,7 @@ public void prepare(
         List<String> addresses = ConfUtils.loadListFromConf(URLFRONTIER_ADDRESS_KEY, stormConf);
 
         // Selected address
-        String address;
+        // String address;
```

Comment:
If it is not needed anymore, we should remove it.

Reply:
I  am probably misreading the code but isn't it used below?

Reply:
It became a class variable (so no local variable needed anymore).

Reply:
Thanks! Spotted it since. Need another coffee

Reply:
Thanks, I'll avoid these:)